obelisk_ws/src/obelisk_msgs/obelisk_control_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(obelisk_control_msgs)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
```

Comment:
Unless ros requires a lower standard, I think we should set this to C++ 17 - there are some nice things in 17 I think we will use (like the filesystem library iirc)

Reply:
done.